frontend/src/router.ts

```diff
@@ -27,5 +28,10 @@ export default new Router({
       name: 'login',
       component: Login,
     },
+    {
+      path: '/register',
+      name: 'register',
+      component: Register,
```

Comment:
```suggestion
      component: () => import(/* webpackChunkName: "register" */ './views/Register.vue'),
```
Look up vue lazy loading. Then you can remove the import at the top of the screen.